bin/push-to-local.sh

```diff
@@ -21,6 +21,7 @@ IMAGES=(
     crunchy-scheduler
     crunchy-upgrade
     crunchy-backrest-restore
+    crunchy-postgres-ha
```

Comment:
@andrewlecuyer crunchy-postgres-gis-ha too?

Reply:
`crunchy-postgres-gis-ha` is now included